lib/job.js

```diff
@@ -193,13 +193,25 @@ class Job extends BaseModel {
 
     /**
      * Return latest build that belong to this job
-     * @param  {Object}   [config]                  Configuration object
-     * @param  {String}   [config.status]           Return latest build with this status
-     * @return {Promise}                            Lastest build
+     * @param  {Object}   [config]              Configuration object
+     * @param  {String}   [config.status]       Return build with this status
+     * @param  {String}   [config.position]     Return build with this position relative to latest build
+     * @return {Promise}                        Lastest build
      */
     getLatestBuild(config = {}) {
-        return this.getBuilds({ status: config.status })
-            .then(latestBuilds => latestBuilds[0] || {});
+        const { position, status } = config;
+        // passing string instead of number to retain old behavior
+        const parsedPoistion = Number(position);
```

Comment:
[`Number`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) could return `NaN`

```
> Number('123')
123
> Number('x123')
NaN
> Number({a: 123})
NaN
```

So, we need to guard and set a default value for it.

Reply:
```
When `parsedPoistion = '0.1'`;

        if (parsedPoistion && parsedPoistion > 0) {
            relativePosition = parsedPoistion;
        }
```

`relativePosition <= (Number(0.01) && Number(0.01) > 0)` would be `0.1`

Reply:
thing like this will just make function return an empty object, we can do a Math.floor but I am not sure if that's what we want

Reply:
it's user error IDK if we want to recover from it at all

Reply:
In that case, just throw exception is good enough.